recipes/picmistandard/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "picmistandard" %}
+{% set version = "0.0.14" %}
+{% set build = 0 %}
+{% set sha256 = "b7eefdae1c43119984226b2df358c86fdeef7495084e47b3575e3d07e790ba30" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/picmi-standard/picmi/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./PICMI_Python --no-deps -vv  # [unix]
+  script: {{ PYTHON }} -m pip install .\PICMI_Python --no-deps -vv  # [win]
```

Maybe we can consolidate these?

```suggestion
  script: {{ PYTHON }} -m pip install PICMI_Python --no-deps -vv
```

---

I tried this but then it will query pypa and complain that there is no `picmi-python` package on it

---

What about this? Basically just trying to get rid of the selectors on a `noarch` package

```suggestion
  script:
    - cd PICMI_Python
    - {{ PYTHON }} -m pip install . --no-deps -vv
```

---

Good idea, thanks!